analytics.py

```diff
@@ -1,12 +1,13 @@
 
 import importlib
 import contextlib
-import datetime
 import os
 from collections import defaultdict
+from sqlalchemy.orm.session import Session
+
 from .model import ExternalIntegration
 from .config import CannotLoadConfiguration
-from sqlalchemy.orm.session import Session
```

Comment:
I made misc cosmetic changes for package imports throughout the app in this branch.